src/components/course_admin/manage_projects/single_project.vue

```diff
@@ -67,7 +68,7 @@
                 </template>
                 <div slot-scope="{item}">
                   <span>
-                    {{item.name ? item.name : ""}}
+                    {{item.name ? item.name : ""}} -
```

Comment:
Same here. It's probably worth writing a utility function that takes in a course and returns a string with name, semester, year